src/components/drawing/index.js

```diff
@@ -442,7 +442,7 @@ drawing.singlePointStyle = function(d, sel, trace, fns, gd) {
                 fill: 'none'
             });
     } else {
-        sel.style('stroke-width', lineWidth + 'px');
+        sel.style('stroke-width', (d.isBlank ? 0 : lineWidth) + 'px');
```

Comment:
Great - tests are passing. Looks like this didn't break @archmoj `isBlank` work from https://github.com/plotly/plotly.js/pull/4056 :tada: